funnel/jobs/jobs.py

```diff
@@ -13,9 +13,12 @@
 
 
 @rq.job('funnel')
-def send_mail_async(sender, to, body, subject):
+def send_mail_async(sender, to, body, subject, file=None):
     with app.app_context():
-        send_mail(sender, to, body, subject)
+        if file is not None:
+            send_mail(sender, to, body, file, subject)
+        else:
+            send_mail(sender, to, body, subject)
```

Comment:
Check if it's ok to pass `file=None` to `send_mail`, then you dont need this if-else block.